src/java/org/apache/cassandra/db/lifecycle/View.java

```diff
@@ -282,7 +287,11 @@ public boolean apply(View view)
             {
                 for (SSTableReader reader : readers)
                     if (view.compacting.contains(reader) || view.sstablesMap.get(reader) != reader || reader.isMarkedCompacted())
+                    {
+                        logger.debug("Refusing to compact {}, already compacting={}, suspect={}, compacted={}", reader,
```

Comment:
I am wonder about terminology here. Is it better to use `refusing` as in the original PR or `rejecting` as in the method comment above? (not an importance, just wondering)

Reply:
Yeah, could be `rejecting, `skipping`, `Cannot compact {}`, `Compacting {} not permitted` ... but I think someone reading the log message will still get the idea.